pkg/util/globalerror/errors.go

```diff
@@ -31,6 +31,8 @@ const (
 	MaxSeriesPerQuery             ID = "max-series-per-query"
 	MaxChunkBytesPerQuery         ID = "max-chunks-bytes-per-query"
 
+	CompactorBlockUploadMaxBlockSizeBytes ID = "compactor-block-upload-max-block-size-bytes"
```

Comment:
Not sure if we need to include "compactor" in the ID... I'd suggest to use feature name only ("block-upload").

Reply:
I removed this for now.